Hardware/Hardware/CardReader/CardReaderEvent.swift

```diff
@@ -1,5 +1,14 @@
 /// The possible events from a connected reader.
 public enum CardReaderEvent {
+    /// The reader begins waiting for input.
+    /// The app should prompt the customer to present a payment method
+    case waitingForInput(String)
+
+    /// Request that a prompt be displayed in the app.
+    /// For example, if the prompt is SwipeCard,
+    /// the app should instruct the user to present the card again by swiping it.
+    case displayMessage(String)
+
     // A card was inserted.
     case cardInserted
```

Comment:
Will `cardInserted` also fire for tapped or swiped cards? If so, please update comment and make it `///` (not `//`) so xcode displays it like the others you are adding